src/hdf5/osw_2ioda.py

```diff
@@ -94,6 +91,7 @@ def main(args):
             print(f'Input (-i option) file: {file_name} does not exist')
             sys.exit()
         logging.debug(f"Reading input file: {file_name}")
+        print(f"Reading input file: {file_name}")
```

Comment:
```suggestion
```

Reply:
we should discuss this, but think this should always be printed so added it intentionally

Reply:
Sure, but why keep two print lines? The line above can be changed to `logging.info` instead of debug.